src/plugins/robot-memory/robot_memory.cpp

```diff
@@ -797,6 +797,7 @@ RobotMemory::mutex_create(const std::string& name)
 	insert_doc.append("_id", name);
 	insert_doc.append("locked", false);
 	try {
+		MutexLocker locker(mutex_);
```

Comment:
```suggestion
		MutexLocker lock(mutex_);
```
This is the more common use case (see ```git grep MutexLocker```). It also eases a later tansition to std::unique_lock (or similar), where we to ever make such a transition.